docs/src/guide/tutorial.jl

```diff
@@ -0,0 +1,298 @@
+# # Tutorial
+#md # [![](https://mybinder.org/badge_logo.svg)](@__BINDER_ROOT_URL__/guide/@__NAME__.ipynb)
+#md # [![](https://img.shields.io/badge/show-nbviewer-579ACA.svg)](@__NBVIEWER_ROOT_URL__/guide/@__NAME__.ipynb)
+
+#nb # DFTK is a Julia package for playing with plane-wave
+#nb # density-functional theory algorithms. In its basic formulation it
+#nb # solves periodic Kohn-Sham equations.
+#
+# This document provides an overview of the structure of the code
+# and how to access basic information about calculations.
+# Basic familiarity with the concepts of plane-wave density functional theory
+# is assumed throughout.
+#
+# For our discussion we will use the classic example of
+# computing the LDA ground state of the silicon crystal.
+# Performing such a calculation roughly proceeds in three steps[^1]:
+#
+# [^1]: Notice that with these parameters the results are still far from convergence in `Ecut` or `kgrid`.
+
+using DFTK
+
+## 1. Define lattice and atomic positions
+a = 10.26  # Silicon lattice constant in Bohr
+lattice = a / 2 * [[0 1 1.];
+                   [1 0 1.];
+                   [1 1 0.]]
+
+## Load HGH pseudopotential for Silicon
+Si = ElementPsp(:Si, psp=load_psp("hgh/lda/Si-q4"))
+
+## Specify type and positions of atoms
+atoms = [Si => [ones(3)/8, -ones(3)/8]]
+
+## 2. Select model and basis
+model = model_LDA(lattice, atoms)
+kgrid = [4, 4, 4]  # k-point grid (Regular Monkhorst-Pack grid)
+Ecut = 7           # kinetic energy cutoff in Hartree
+basis = PlaneWaveBasis(model, Ecut; kgrid=kgrid)
+
+## 3. Run the SCF procedure to obtain the ground state
+scfres = self_consistent_field(basis, tol=1e-8);
+
+# Notice that DFTK by default applies the convergence tolerance `tol`
+# to the energy difference, such that the norm in the density difference
+# is not yet converged to 8 digits.
+#
+# For all quantities DFTK uses
+# [atomic units](https://en.wikipedia.org/wiki/Hartree_atomic_units),
+# where lengths are in Bohr, energies in Hartree and so on.
+# For conversion to other units, DFTK provides conversion factors
+# in the `DFTK.units` module.
+# The convention is that the conversion factors convert *into* atomic units,
+# therefore:
+
+import DFTK.units: eV
+10eV      # 10eV in Hartree
+#-
+1.2 / eV  # 1.2 Hartree in eV
+
+# Computations take place in the unit cell of a lattice defined by a ``3 × 3``
+# matrix (`model.lattice`) with lattice vectors as columns.
+# The reciprocal lattice `model.recip_lattice`,
+# that is the lattice of Fourier coefficients of functions
+# with the periodicity of the lattice) is defined by the matrix
+# ```math
+# B = 2\pi A^{-T} = 2π {A^{T}}^{-1}
+# ```
+# where ``A`` is the unit cell. As a result:
```

Comment:
Confusion here between unit cell and lattice vectors. Also link A and lattice, and B and recip_lattice